feature-rich/package.json

```diff
@@ -1,141 +1,143 @@
+		"@ckeditor/ckeditor5-alignment": "39.0.1",
+		"@ckeditor/ckeditor5-autoformat": "39.0.1",
+		"@ckeditor/ckeditor5-basic-styles": "39.0.1",
+		"@ckeditor/ckeditor5-block-quote": "39.0.1",
+		"@ckeditor/ckeditor5-ckbox": "39.0.1",
+		"@ckeditor/ckeditor5-ckfinder": "39.0.1",
+		"@ckeditor/ckeditor5-clipboard": "39.0.1",
+		"@ckeditor/ckeditor5-cloud-services": "39.0.1",
+		"@ckeditor/ckeditor5-code-block": "39.0.1",
+		"@ckeditor/ckeditor5-core": "39.0.1",
+		"@ckeditor/ckeditor5-dev-translations": "38.3.1",
+		"@ckeditor/ckeditor5-dev-utils": "38.3.1",
+		"@ckeditor/ckeditor5-editor-classic": "39.0.1",
+		"@ckeditor/ckeditor5-engine": "39.0.1",
+		"@ckeditor/ckeditor5-enter": "39.0.1",
+		"@ckeditor/ckeditor5-essentials": "39.0.1",
+		"@ckeditor/ckeditor5-export-pdf": "39.0.1",
+		"@ckeditor/ckeditor5-export-word": "39.0.1",
+		"@ckeditor/ckeditor5-find-and-replace": "39.0.1",
+		"@ckeditor/ckeditor5-font": "39.0.1",
+		"@ckeditor/ckeditor5-heading": "39.0.1",
+		"@ckeditor/ckeditor5-highlight": "39.0.1",
+		"@ckeditor/ckeditor5-horizontal-line": "39.0.1",
+		"@ckeditor/ckeditor5-html-embed": "39.0.1",
+		"@ckeditor/ckeditor5-html-support": "39.0.1",
+		"@ckeditor/ckeditor5-image": "39.0.1",
+		"@ckeditor/ckeditor5-import-word": "39.0.1",
+		"@ckeditor/ckeditor5-indent": "39.0.1",
+		"@ckeditor/ckeditor5-language": "39.0.1",
+		"@ckeditor/ckeditor5-link": "39.0.1",
+		"@ckeditor/ckeditor5-list": "39.0.1",
+		"@ckeditor/ckeditor5-media-embed": "39.0.1",
+		"@ckeditor/ckeditor5-mention": "39.0.1",
+		"@ckeditor/ckeditor5-page-break": "39.0.1",
+		"@ckeditor/ckeditor5-paragraph": "39.0.1",
+		"@ckeditor/ckeditor5-paste-from-office": "39.0.1",
+		"@ckeditor/ckeditor5-remove-format": "39.0.1",
+		"@ckeditor/ckeditor5-select-all": "39.0.1",
+		"@ckeditor/ckeditor5-special-characters": "39.0.1",
+		"@ckeditor/ckeditor5-style": "39.0.1",
+		"@ckeditor/ckeditor5-table": "39.0.1",
+		"@ckeditor/ckeditor5-theme-lark": "39.0.1",
+		"@ckeditor/ckeditor5-typing": "39.0.1",
+		"@ckeditor/ckeditor5-ui": "39.0.1",
+		"@ckeditor/ckeditor5-undo": "39.0.1",
+		"@ckeditor/ckeditor5-upload": "39.0.1",
+		"@ckeditor/ckeditor5-utils": "39.0.1",
+		"@ckeditor/ckeditor5-widget": "39.0.1",
+		"@ckeditor/ckeditor5-word-count": "39.0.1",
+		"@ckeditor/ckeditor5-template": "39.0.1",
+		"@ckeditor/ckeditor5-document-outline": "39.0.1",
+		"@ckeditor/ckeditor5-format-painter": "39.0.1",
+		"@ckeditor/ckeditor5-slash-command": "39.0.1",
+		"@webspellchecker/wproofreader-ckeditor5": "2.3.1",
+		"css-loader": "^5.2.7",
+		"postcss": "^8.4.19",
+		"postcss-loader": "^4.3.0",
+		"raw-loader": "^4.0.2",
+		"style-loader": "^2.0.0",
+		"terser-webpack-plugin": "^4.2.3",
+		"webpack": "^5.75.0",
+		"webpack-cli": "^4.10.0",
+		"webpack-dev-server": "^4.11.1"
+	},
+	"scripts": {
+		"build": "webpack --mode production",
+		"build-dev": "webpack --mode development",
+		"start": "webpack serve --open /index.html --mode development"
+	},
+	"resolutions": {
```

Comment:
https://github.com/ckeditor/ckeditor5-demos/issues/29#issuecomment-1721083665 Resolutins are not necessary any more.